iceoryx_dds/include/iceoryx_dds/gateway/gateway_config.hpp

```diff
@@ -24,12 +24,17 @@ namespace iox
 namespace dds
 {
 ///
-/// @brief Generic configuration for DDS gateways.
+/// @brief Generic configuration for gateways.
```

Comment:
Should this be moved out of `/iceoryx_dds/*` if it can be reused for other gateways?

Reply:
Will address this in the future.